haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -6,14 +6,14 @@ COPYRIGHT="2011 Alexey Burshtein
 	2001-2002 Kevin H. Patterson"
 LICENSE="MIT
 	BSD (3-clause)"
-REVISION="1"
-srcGitRev="2fd70537847d3b9712e96ebbc4ff37d5e961b480"
+REVISION="2"
+srcGitRev="027e4cbdc0340135406624742fa8405388725e46"
 SOURCE_URI="https://github.com/HaikuArchives/Eventual/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="229cd7fa206da0606d32fc0db1f44a5eee20f0e8428e3dec141e4aab272ead68"
+CHECKSUM_SHA256="b0cf1dafa022930ebd4c2fddeeb79271f6038267ea77542afc7931dc4463ae08"
 SOURCE_FILENAME="Eventual-$portVersion-$srcGitRev.tar.gz"
 SOURCE_DIR="Eventual-$srcGitRev"
 
-ARCHITECTURES="x86_gcc2 ?x86_64"
+ARCHITECTURES="x86_gcc2 !x86_64"
```

Comment:
Maybe disable the architectures until the bug if fixed?
```diff
- ARCHITECTURES="x86_gcc2 !x86_64"
+ ARCHITECTURES="?x86_gcc2 ?x86_64"
```

Reply:
I confirmed that Eventual could not be built on x86_64. Fixing it would involve changing a data structure in the code base without breaking x86_gcc2.

The wobbling bug along with another [bug](https://github.com/haikuports/haikuports/pull/2087#discussion_r160406850) were there before the original recipe. They don't prevent the applications from starting and being explored, but I will add the question mark if you insist. :)

Reply:
Let's wait for @scottmc :) Any take on this?

Reply:
Also created an issue for the other bug :)